src/gui/render/opengl_renderer.cpp

```diff
@@ -72,9 +73,19 @@ SDL_Window* OpenGlRenderer::CreateSdlWindow(const int x, const int y,
 	SDL_GL_SetAttribute(SDL_GL_DEPTH_SIZE, 0);
 	SDL_GL_SetAttribute(SDL_GL_DOUBLEBUFFER, 1);
 
+#ifdef USE_DEBUG_CONTEXT
+	SDL_GL_SetAttribute(SDL_GL_CONTEXT_FLAGS, SDL_GL_CONTEXT_DEBUG_FLAG);
+
+	// Request an OpenGL 4.3 core profile context (debug output became a
+	// core part of OpenGL since version 4.3). Note the macOS doesn't seem
+	// to support debug contexts at all.
+	SDL_GL_SetAttribute(SDL_GL_CONTEXT_MAJOR_VERSION, 4);
+	SDL_GL_SetAttribute(SDL_GL_CONTEXT_MINOR_VERSION, 3);
```

Comment:
Maybe try OpenGL 4.1? It looks like 4.1 is the latest version that Mac supports.